tests/build_/test_build_request.py

```diff
@@ -1163,6 +1163,7 @@ def test_render_optional_plugins(self, tmpdir):
             'source_registry_uri': 'source_registry_uri',
             'sources_command': 'sources_command',
             'vendor': 'vendor',
+            'equal_labels': 'equal_labels',
```

Comment:
Maybe make this a bit more complicated?

Also needs an assert further down to check it was unpacked and repacked correctly, like the other vars.